hw/top_darjeeling/top_pkg.core

```diff
@@ -0,0 +1,22 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# XXX: This name is currently required as global identifier until we have
+# support for "interfaces" or a similar concept.
+# Tracked in https://github.com/olofk/fusesoc/issues/235
+name: "lowrisc:constants:top_pkg"
```

Comment:
Do those core files need to specify a virtual core?

Reply:
Right, with virtual core support this core file would have the name suggested above and specify `lowrisc:constants:top_pkg` as virtual core.

Dependent cores then either have `lowrisc:constants:top_pkg` -- if they don't depend on a _specific_ top -- or `lowrisc:constants:top_darjeeling_top_pkg` -- if they do depend on a specific top -- in their dependencies.

We don't have proper support for virtual cores just yet, so I leave it to you whether you want to do this in this or in a follow-up PR.

Reply:
Let's do that (and the other Todo) in a follow up-PR to unblock multitop SW, which depends on that change. CI is green (so far it can), so let's get that in :)